File: problem02.py
T=int(input())

while(T!=0):
    if T==100:
        break
    a=input()
    b=len(a)
    if b>100:
        break
    c=int(a)
    if c%2==0:
        print("even")
    else:
        print("odd")
    T=T-1

File: problem02.c
#include<stdio.h>
#include<string.h>

int main()
{
    int x,a,b,c;
    char d[101],e;
    scanf("%d",&x);
    while(x--)
    {
    scanf("%s",d);
    b=strlen(d);
    e=d[b-1];
    c=(int)e-48;
    if (c%2==0)
    {
        printf("even\n");
    }
    else
    {

        printf("odd\n");
    }

    }

}

